.storybook/preview.js

```diff
@@ -1,19 +1,23 @@
 import GlobalStyle from '../src/components/GlobalStyle'
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
-
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
```

Comment:
Is this the right import? i only know 'react-router-dom'

Reply:
The link came automatic after I did the import.